packages/playground/worker/__tests__/worker.spec.ts

```diff
@@ -72,4 +72,16 @@ if (isBuild) {
     expect(content).toMatch(`(window.URL||window.webkitURL).createObjectURL`)
     expect(content).toMatch(`window.Blob`)
   })
+
+  test('worker need bundle', () => {
+    fs.readdirSync(assetsDir)
+      .filter(
+        (file) =>
+          file.includes('url-worker') || file.includes('url-shared-worker')
+      )
+      .forEach((file) => {
+        const content = fs.readFileSync(path.resolve(assetsDir, file), 'utf-8')
+        expect(content.startsWith('(function(){')).toBe(true)
```

Comment:
This assertion doesn't look very future-proof to me.

But if we don't find better ways to test it, I'm open to merging this PR first and refactoring the tests later.